driver-core/src/main/com/mongodb/internal/operation/OperationHelper.java

```diff
@@ -479,15 +480,14 @@ static boolean canRetryRead(final ServerDescription serverDescription, final Con
                                                           final ConnectionSource source, final Connection connection, final int batchSize) {
         return new QueryBatchCursor<T>(OperationHelper.<T>cursorDocumentToQueryResult(cursorDocument,
                 source.getServerDescription().getAddress()),
-                0, batchSize, 0, decoder, source, connection);
+                0, batchSize, 0, decoder, null, source, connection);
```

Comment:
Why `null` instead of `comment`, like it is for async just below?

Reply:
Good catch :+1:

Reply:
Can I deduce that there is no test coverage for this scenario?

Reply:
No - it would be for a getMore on a list collection / list indexes operation.